.babelrc.json

```diff
@@ -10,7 +10,12 @@
       }
     ],
     "@babel/preset-typescript",
-    "@babel/preset-react"
+    [
+      "@babel/preset-react",
+      {
+        "runtime": "automatic"
+      }
+    ]
```

Comment:
What is the case for setting runtime explicitly here? I believe automatic is the default

Reply:
That's a weird one, without explicitly setting runtime to `automatic` it results in all the stories crashing
<img width="1046" alt="image" src="https://github.com/aragon/ods/assets/172602066/20b14375-d658-4df9-b589-c7f4e4bb5518">